src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -61,6 +69,10 @@ const ColorRampSelector: React.FC<ColorRampSelectorProps> = (propsInput): ReactE
   const props = { ...defaultProps, ...propsInput } as Required<ColorRampSelectorProps>;
   const theme = useContext(AppThemeContext);
 
+  const colorRampsToDisplay = props.colorRampsToDisplay ?? Array.from(props.knownColorRamps.keys());
+  const categoricalPalettesToDisplay =
+    props.categoricalPalettesToDisplay ?? Array.from(props.knownCategoricalPalettes.keys());
+
```

Comment:
Same question as above, if the known ramps and palettes are truly legacy and meant to be accessed only by URL, why provide them as a fallback?